StripeFinancialConnections/StripeFinancialConnections/Source/Native/AccountPicker/AccountPickerFooterView.swift

```diff
@@ -23,6 +23,7 @@ final class AccountPickerFooterView: UIView {
         NSLayoutConstraint.activate([
             linkAccountsButton.heightAnchor.constraint(equalToConstant: 56)
         ])
+        linkAccountsButton.accessibilityIdentifier = "account_picker_link_accounts_button"
```

Comment:
the accessibility identifiers help to make tests more stable because we will be changing copy in the future